internal/services/netapp/netapp_volume_resource_test.go

```diff
@@ -1359,10 +1410,26 @@ resource "azurerm_subnet" "test-delegated" {
 }
 
 resource "azurerm_subnet" "test-non-delegated" {
-  name                 = "acctest-Non-Delegated-Subnet-%[1]d"
+  name                 = "acctest-Non-Delegated-Subnet-%[2]d"
   resource_group_name  = azurerm_resource_group.test.name
   virtual_network_name = azurerm_virtual_network.test.name
   address_prefixes     = ["10.6.0.0/24"]
 }
-`, data.RandomInteger)
+`, r.templateProviderFeatureFlags(), data.RandomInteger)
+}
+
+func (NetAppVolumeResource) templateProviderFeatureFlags() string {
+	return `
+provider "azurerm" {
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
+    netapp {
+      prevent_volume_destruction = false
+      delete_backups_on_backup_vault_destroy = true
+	  } 
```

Comment:
I think this might be causing issues with the unit test by not being spaced correctly